docs/networkpolicy-recommendation.md

```diff
@@ -0,0 +1,148 @@
+# Network Policy Recommendation
+
+## Table of Contents
+
+<!-- toc -->
+- [Introduction](#introduction)
+- [Installation](#installation)
+- [Perform NetworkPolicy Recommendation](#perform-networkpolicy-recommendation)
+  - [Run a policy recommendation job](#run-a-policy-recommendation-job)
+  - [Check the status of a policy recommendation job](#check-the-status-of-a-policy-recommendation-job)
+  - [Retrieve the result of a policy recommendation job](#retrieve-the-result-of-a-policy-recommendation-job)
+<!-- /toc -->
+
+## Introduction
+
+Theia NetworkPolicy Recommendation recommends the NetworkPolicy configuration to secure Kubernetes
+network and applications. It analyzes the network flows collected by
+[Grafana Flow Collector](network-flow-visibility.md#grafana-flow-collector) to generate
+[Kubernetes NetworkPolicies](https://kubernetes.io/docs/concepts/services-networking/network-policies/)
+or [Antrea NetworkPolicies](https://github.com/antrea-io/antrea/blob/main/docs/antrea-network-policy.md).
+This feature assists cluster administrators and app developers in securing their applications
+according to Zero Trust principles.
+
+## Installation
+
+Theia's NetworkPolicy Recommendation capablity leverages Antrea Flow Exporter and Flow Aggreator.
+Before leveraging this feature, follow this [doc](https://github.com/antrea-io/antrea/blob/main/docs/network-flow-visibility.md)
+to ensure they are deployed. In addition, ensure `podLabels: true` is set in [Flow Aggregator Configuration](https://github.com/antrea-io/antrea/blob/main/docs/network-flow-visibility.md#configuration-1),
+as Theia's NetworkPolicy Recommendation feature needs to process pod label information.
+
+Then, please follow these [deployment steps](network-flow-visibility.md#deployment-steps)
+to deploy Theia.
+
+Commet out following lines since user won't need to deploy Spark Operator separately after PR#31 merged.
```

Comment:
I think the Installation section might become obsolete considering spark-operator is also being adding to flow-visibility.yaml?

I would probably list "pre-requisites" here, and namely:
- flow exporter must be enabled (which is a pre-requisite for all network visibility)
- flow aggregator must store pod label

It is also true that we want to have a "hard way" installation story (the one below), but it should be - in my opinion - together with flow visibility installation info

Reply:
`Prerequisite` sounds good to me, updated this doc first, and will change the `flow visibility` doc after we finalized the overall install story.